site/en/blog/deprecating-web-sql/index.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Deprecating Web SQL
+description: >
+  The Web SQL Database standard was first proposed in April 2009 and abandoned in November 2010.
+  While it was implemented in WebKit and remained active in Blink, Gecko never implemented this
+  feature and WebKit deprecated it in 2019. The W3C encourages those needing web databases to adopt
+  Web Storage or Indexed Database. Web SQL has been deprecated and removed for third-party contexts
+  in Chromium 97. Now Chromium 105 deprecates Web SQL in insecure contexts. The complete removal of
+  the feature is planned for Chromium 107.
+authors:
+  - thomassteiner
+date: 2022-08-02
+# updated: 2022-08-02
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/yUp8lfaCt4EmxmVei3lj.jpg
+alt: Filing cabinet symbolizing a database.
+---
+
+The [Web SQL Database](https://www.w3.org/TR/webdatabase/) proposal was first
+[introduced in April 2009](https://www.w3.org/TR/2009/WD-webdatabase-20091222/) and
+[abandoned in November 2010](https://www.w3.org/TR/webdatabase/#status-of-this-document). While it
+was implemented in the WebKit browser engine (that powers Safari and early versions of Chrome) and
+remained active in the Blink engine (that powers Chrome after the switch from WebKit), the Gecko
+engine (that powers Firefox) never implemented this feature and
+[WebKit deprecated it in 2019](https://lists.webkit.org/pipermail/webkit-dev/2019-November/030968.html).
+The World Wide Web Consortium (W3C)
+[encourages](https://www.w3.org/TR/webdatabase/#:~:text=The%20Web%20Applications%20Working%20Group%20continues%20work%20on%20two%20other%20storage%2Drelated%20specifications%3A%20Web%20Storage%20and%20Indexed%20Database%20API.)
+those needing web databases to adopt
+[Web Storage](https://developer.mozilla.org/docs/Web/API/Web_Storage_API) or
+[Indexed Database](https://developer.mozilla.org/docs/Web/API/IndexedDB_API/Using_IndexedDB).
+
+## Web SQL deprecation steps
+
+Now, in the Chromium project that browsers like Chrome and Edge are built upon, we're taking steps
+to deprecate Web SQL for good, too:
+
+- Web SQL has been deprecated and removed for **third-party contexts** in **Chromium&nbsp;97**.
+- Web SQL access in **insecure contexts** is deprecated as of **Chromium&nbsp;105**.
+- The **complete removal** of the feature is planned for **Chromium&nbsp;107**.
+
+## Reasons for deprecating Web SQL
+
+### Sustainability and security concerns
+
+The Web SQL specification cannot be implemented sustainably. The last version of the standard
```

Comment:
I think it would be nice to say upfront that we're not planning to just remove WebSQL. In fact, we're planning to replace it with something that will be maintained by the open-source community, served as a package that can be updated at will, without the burden of introducing fixes and new features into browsers.

What's more, we're hoping that this example will help a new ecosystem of open-source databases will flourish! No more will browsers keep trying to create storage solutions that don't quite work right...

Reply:
I've added text based on your comment to the introduction.